app/middlewares/errors.js

```diff
@@ -8,6 +8,7 @@ const statusCodes = {
   [errors.BOOK_NOT_FOUND]: 404,
   [errors.SAVING_ERROR]: 400,
   [errors.DATABASE_ERROR]: 503,
+  [errors.MISSING_VALUE]: 404,
```

Comment:
Is not a 404 message, remember that is a `Not found` this is more like a `Bad Request - 400`